pkg/scheduler/scheduler.go

```diff
@@ -224,16 +227,15 @@ func (s *admissionStatus) IsError() bool {
 // AsStatus wraps an error in a admissionStatus.
 func AsStatus(err error) *admissionStatus {
 	return &admissionStatus{
-		reasons: []string{err.Error()},
-		err:     err,
+		err: err,
 	}
 }
 
 // assignFlavors calculates the flavors that should be assigned to this entry
 // if admitted by this clusterQueue, including details of how much it needs to
 // borrow from the cohort.
-// It returns whether the entry would fit. If it doesn't fit, the object is
-// unmodified.
+// It returns admissionStatus indicating whether the entry fits. If it doesn't fit,
+// the object is unmodified.
```

Comment:
Let's be more clear:

```suggestion
// the entry is unmodified.
```